PluginCore/ScintillaNet/Events.cs

```diff
@@ -19,8 +19,8 @@ namespace ScintillaNet
     public delegate void NeedShownHandler(ScintillaControl sender, int position, int length);
     public delegate void UserListSelectionHandler(ScintillaControl sender, int listType, string text);
     public delegate void URIDroppedHandler(ScintillaControl sender, string text);
-    public delegate void DwellStartHandler(ScintillaControl sender, int position);
-    public delegate void DwellEndHandler(ScintillaControl sender ,int position);
+    public delegate void DwellStartHandler(ScintillaControl sender, int position, int x, int y);
```

Comment:
Sadly @wise0704 is busy again with personal stuff like university. I'll try to make tests covering that soon.